products/bigqueryconnection/terraform.yaml

```diff
@@ -21,7 +21,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       cloudSql.credential.password: !ruby/object:Overrides::Terraform::PropertyOverride
         sensitive: true
     id_format: "{{name}}"
-    import_format: ["{{name}}"]
+    import_format: ["projects/{{project}}/locations/{{location}}/connections/{{connection_id}}", "{{project}}/{{location}}/{{connection_id}}", "{{location}}/{{connection_id}}"]
```

Comment:
This looks about right based on other `import_format`s.

Should `id_format` also be updated? It looks like the id_format is sometimes the long form and sometimes `{{name}}` so I'm not clear on whether that's intentional or legacy.

Is there a way to add a test to ensure that this works properly?

Reply:
The `id_format` is set after apply, so it's safe to use `name`, which is an output field.  That's usually the difference - sometimes `name` is an output and sometimes it's an input.  When it's an output, it should be the ID, when it's an input, ID format should be the long form.  Not to say we haven't made occasional mistakes along the way, but that *should* be the difference, at least.

Reply:
A test... well, there should be, right?  But the thing is, the existing test should have caught this.  I'm going to spend a little extra time trying to figure out why it didn't.

Reply:
ah.  we don't actually do import state verify in the tests!  that's why!  Let me fix that.

Reply:
Okay!  Finally done.  That was a tough one but I got it once I got started, haha.